Cleanup data directory under forced photometry

jkrick opened the pull request:
This will close #489 

It would be great if someone could double check that multiband_photometry runs for them, I am not entirely confident that my work on this is perfect.

jkrick committed:
Remove data/multiband_phot.fits

jkrick committed:
Remove data/Chandra/COSMOS_chandra.fits

jkrick committed:
removing Galex 001star

jkrick committed:
removing Galex/COSMOS..fits

jkrick committed:
removing IRAC/.nfs

jkrick committed:
removing IRAC/001...fits

jkrick committed:
removing IRAC table_irsa...csv

jkrick committed:
removing IRAC/IRAC..EXTPRF.fits

jkrick committed:
Revert "Remove data/Chandra/COSMOS_chandra.fits"

This reverts commit 2a20359405a50a958b8e60773b0b8744b5c5f8aa.

jkrick committed:
Revert "Remove data/multiband_phot.fits"

This reverts commit 9d13e714fb2badecbda54cc05886c96e392d098a.

troyraen reviewed (commented):
Can Chandra/chandra_multiband.fits also be removed?

jkrick committed:
removed/chandra/chandra_multiband.fits

jkrick commented:
> Can Chandra/chandra_multiband.fits also be removed?

yes, I removed it.  Also changed the runtime to reflect what it takes on small instance with no previously downloaded data (13 min.).  

@troyraen please approve and merge when ready.

jkrick committed:
changed runtime

troyraen reviewed (approved):
Thanks! I tested it as well and it ran without error and in about the same amount of time as what's now in the runtime statement.
